doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/runtime-manager-strategies-con.adoc

```diff
@@ -0,0 +1,60 @@
+[id='runtime-manager-strategies-con_{context}']
+= Runtime manager strategies
+
+The runtime manager supports the following strategies for managing KIE sessions.
+
+*Singleton strategy*: instructs `RuntimeManager` to maintain a single instance of `RuntimeEngine` (and in turn single instance of `KieSession` and `TaskService`). Access to the runtime engine is synchronized and, therefore, thread safe, although it comes with a performance penalty due to synchronization.
+
+ifdef::DROOLS,JBPM,OP[]
+This strategy is similar to what was available by default in jBPM version 5.x. 
+endif::DROOLS,JBPM,OP[]
+Use this strategy for simple use cases.
+
+It has the following characteristics that can be important for a usage scenario:
+
+* It has a small memory footprint - single instance of the runtime engine and the task service.
+* It is simple and compact in design and usage.
+* It is a good fit for low to medium load on the {PROCESS_ENGINE} because of synchronized access.
+* Because of the single KieSession instance, all state objects (such as facts) are directly visible to all process instances and vice versa.
+* The strategy is not contextual. When you retrieve instances of RuntimeEngine from a singleton RuntimeManager, you do not need to take the `Context` instance into account. Usually, you can use EmptyContext.get() as the context, although a null argument is acceptable as well
```

Comment:
Again for parallelism, "It is not contextual."  Also, in this sentence need to unicode `RuntimeEngine`, `RuntimeManager` and `EmptyContext.get()`.

Reply:
I did not find perfect parallelism possible without significant contortion of the language, however I made sure all lines have either "it" or "this strategy/the strategy".